test/cypress/support/commands.js

```diff
@@ -165,11 +224,21 @@ Cypress.Commands.add('giAcceptGroupInvite', (invitationLink, {
   isLoggedIn,
   inviteCreator,
   displayName,
-  actionBeforeLogout
+  actionBeforeLogout,
+  bypassUI
 }) => {
+  if (bypassUI && !isLoggedIn) {
+    // TODO - Create bypass action to join group. Until then
+    // do the other way around: signup before visiting the page.
+    // Then, the invitation is automatically accepted.
+    cy.giSignup(username, { bypassUI })
+    // use an hardcoded wait so indexedDB is synced before cy.visit()
+    cy.wait(400) // eslint-disable-line
```

Comment:
I thought we no longer need hardcoded waits...?

Reply:
We need it here because we do cy.visit() immediately after a bypass-ui action.

But nevermind, I removed this code to avoid confusion. We'll do the "join group" bypass-ui action in a proper way (without wait) on another MR. This was just a middle solution but it's not worth the confusion.